<!-- Launch file for the assigner -->
<launch>
  <arg name="robot_index" default=""/>
  <param name="/use_sim_time" value="true"/>

  <node pkg="frontier_exploration" type="decent_robot_assigner.py" name="robot_assigner_$(arg robot_index)" output="screen">
  <param name="map_topic" value="/map"/>
  <param name="info_radius" value="0.1"/> 
  <param name="frontiers_topic" value="/filtered_frontiers"/> 
  <param name="n_robots" value="2"/>
  <param name="delay_after_assignement" value="0.2"/>
  <param name="rate" value="1"/>
  <param name="robot_number" value="$(arg robot_index)"/>
  </node>

</launch>

